templates/gp-templates-overrides/translation-row-editor-meta-status.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Template for the meta section of the editor row in a translation set display
+ *
+ * @package    GlotPress
+ * @subpackage Templates
+ */
+
+?>
+<dl>
+	<?php gp_tmpl_load( 'translation-row-editor-meta-feedback', get_defined_vars(), dirname( dirname( __FILE__ ) ) ); ?>
```

Comment:
I think this should be without a directory, so that it will load potential overrides.
```suggestion
	<?php gp_tmpl_load( 'translation-row-editor-meta-feedback', get_defined_vars() ); ?>
```